### ISSUE TITLE
C#. Failure when trying to compare arrays

### ISSUE THREAD

**Issue by fhgaha** [AlexDarigan/WAT#350]
Test like this
```
[Test]
public void MyTest()
{
	Assert.IsEqual(new int[] { 0 }, new int[] { 0 });
}
```
returns log like this
<img width="528" alt="hru8iCooIp" src="https://user-images.githubusercontent.com/84946584/184516848-d7d7258c-4eeb-406a-b785-193e3724aeb5.png">

This test should pass.
Or maybe some method to compare collections should be implemented.

**Comment by AlexDarigan** [comment]
Will add something like IsContentEquals.

**Comment by fhgaha** [comment]
My bad. Code like this 
`        static void Main(string[] args)
        {
            Console.WriteLine(new int[] { 0 }.Equals(new int[] { 0 }));
            Console.WriteLine(new int[] { 0 } == new int[] { 0 });
        }`
returns `False False` too. I guess you should not compare collections like this.

**Comment by AlexDarigan** [comment]
Resolved by #353
